src/mixxx.cpp

```diff
@@ -451,7 +469,15 @@ void MixxxMainWindow::finalize() {
     Timer t("MixxxMainWindow::~finalize");
     t.start();
 
-    // Save the current window state (position, maximized, etc)
+    UserSettingsPointer pConfig = m_pSettingsManager->settings();
+    int inhibit = pConfig->getValue<int>(ConfigKey("[Config]","InhibitScreensaver"));
+    mixxx::ScreenSaverPreference inhiPref = mixxx::ScreenSaverPreference(inhibit);
```

Comment:
You can keep it, if you like. 

I have just looked in 
https://google.github.io/styleguide/cppguide.html#Casting
In this case your enum class has no constructor so I think static_cast<> it the best choice here. 
We only user auto rarely, except if it helps not to repeat ourselves like in this case.